### PR TITLE
Docs Landing Page Restructure (first pass)

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

Without restructuring the folders and content itself (that'll be my next step), this was a first pass at surfacing some of the important topic and trying to align with the major headlines flagged in https://github.com/flowforge/website/issues/842

## Related Issue(s)

https://github.com/flowforge/website/issues/842

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] Documentation has been updated

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,32 +2,55 @@
...
-services. FlowForge adds to Node-RED; collaborative development, management of
+services.
+
+FlowForge adds to Node-RED; collaborative development, management of
 remote deployments, support for DevOps delivery pipelines, and the ability to
 host Node-RED applications on FlowForge. FlowForge is the DevOps platform for
 Node-RED application development and delivery.
 
-## Overview
+## Using FlowForge
```
Comment (ZJvandeWeg) [inline docs/README.md:17]: Is this first time users we're targetting?

```suggestion
## Getting started
```

File: docs/README.md [hunk 2]
```diff
@@ -2,32 +2,55 @@
...
 
- - [FlowForge Concepts](./user/concepts.md) - understanding the platform
- - [FlowForge Cloud](./cloud/) - details specific to the hosted service
- - [Administrator documentation](./admin/) - how to administer the platform
- - [User documentation](./user/) - how to use the platform
+ - [FlowForge Concepts](./user/concepts.md) - Understanding the core terminology and concepts that make up the FlowForge platform.
```
Comment (ZJvandeWeg) [inline docs/README.md:19]: Styling wise, what do you think about using tables with bigger cells? I've tried this in https://deploy-preview-935--flowforge-website.netlify.app/handbook/ and found it appealing.

File: docs/README.md [hunk 3]
```diff
@@ -2,32 +2,55 @@
...
+
+## Managing FlowForge
 
 - [Installing FlowForge](./install/) - how to install the platform
-- [Upgrading FlowForge](./upgrade/) - how to upgrade the platform
-- [Install the Device Agent](./user/devices.md) - deploy flows to remote devices
+- [Administrator documentation](./admin/) - how to administer the platform
 
-## Upgrading FlowForge
+## Upgrading & Billing
```
Comment (ZJvandeWeg) [inline docs/README.md:42]: Billing is FF Cloud specific I think? 

The two sections above we could rename to `FlowForge Cloud` and `On-premise` maybe, and merge these links to those sections?

File: docs/README.md [hunk 4]
```diff
@@ -2,32 +2,55 @@
...
 
 - [Upgrade your FlowForge Instance](./upgrade/README.md)
+- [Upgrading to Premium](./upgrade/) - how to upgrade the platform
 - [From Open Source to Premium](./upgrade/open-source-to-premium.md)
+
+ ## Contributing to FlowForge
+ - [Useful Information](./contribute/#contributing-to-flowforge/) - Learn the foundational concepts of how FlowForge is built & structured. 
+ - [Development Setup](./contribute/#development-setup/) - Configure your local development environment to contribute to FlowForge.
+ - [Testing](./contribute/#testing/) - Understand our testing philosophy at FlowForge.
```
Comment (ZJvandeWeg) [inline docs/README.md:51]:
```suggestion
 - [Testing](./contribute/#testing/) - Understand our testing philosophy at FlowForge.
 
```

File: docs/README.md [hunk 5]
```diff
@@ -2,32 +2,55 @@
 navTitle: Documentation
 ---
 
-# FlowForge Docs
+# Welcome to FlowForge
 
 FlowForge helps Node-RED developers deliver applications in a more reliable,
```
[the remark this answers is not in the document]
Comment (joepavitt) [reply docs/README.md:7]: yeah, I left it in as was mostly focussed on structuring/grouping at the moment, but was tempted/happy to remove it too.

File: docs/README.md (hunk 1 above)
Comment (joepavitt) [reply docs/README.md:17]: Yep, that's fair.

File: docs/README.md (hunk 2 above)
Comment (joepavitt) [reply docs/README.md:19]: I really liked it. I held off on this, as I had already started the work by the time I saw it. Happy to do that tabling as a follow up PR once we've settled on ordering/grouping here.

File: docs/README.md (hunk 3 above)
Comment (joepavitt) [reply docs/README.md:42]: Yeah I grouped them together as they both covered money-based things. Upgrading > pay us money. Billing also seemed to cover details on how we bill customers generally, not just for FF Cloud?

File: docs/README.md (hunk 4 above)
Comment (joepavitt) [reply docs/README.md:51]: have you changed something here? They look identical?

Comment (ZJvandeWeg) [reply docs/README.md:51]: I've added a new line

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**